autopush_rs/src/client.rs

```diff
@@ -730,6 +730,19 @@ where
             .send()
             .ok();
 
+        // If there's any notifications in the queue, move them to our unacked direct notifs
+        webpush.rx.close();
+        let rx_iter = webpush.rx.wait();
+        for msg in rx_iter {
+            match msg {
+                Ok(ServerNotification::CheckStorage) => continue,
+                Ok(ServerNotification::Notification(notif)) => {
+                    webpush.unacked_direct_notifs.push(notif.clone());
```

Comment:
is this clone necessary? seems like we own it once it comes out of the queue

Reply:
Indeed, removing.